app/src/main/res/drawable/key_background.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@drawable/button_pressed" android:state_pressed="true" />
+    <item android:drawable="@drawable/button_pressed" android:state_focused="true" />
+    <item android:drawable="@drawable/button_pressed" android:state_hovered="true" />
```

Comment:
I think that the change is very, very small that people are using a cursor on their smartphone